notebooks/langchain/langchain-vector-store-using-elser.ipynb

```diff
@@ -4,11 +4,11 @@
    "cell_type": "markdown",
```

Comment:
you also need to specify the name of the model to use it from the default elser v1
```
    strategy=ElasticsearchStore.SparseVectorRetrievalStrategy(model_id=".elser_model_2")
```
double check the model id, im guessing here.

Reply:
Good catch. Will update 👍